docs/content/tooling/coverage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Coverage
+description: How to generate a code coverage report
+status: Stable
+source: https://github.com/xedi/sdk-js/tree/master/testing
+---
+import {Text, Flash, BorderBox} from '@primer/components'
+import {InfoIcon} from '@primer/styled-octicons'
+import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code'
+import {Link} from 'gatsby';
+
+When writing tests it can often prove difficult to ensure we have sufficient coverage across all our code. 
+In this section we will look at using a coverage report to allow us to determine if further testing is required.
+
+## Generating a report
+
+It is first important to ensure the tests are passing and to a high standard. read the <Link to="/tooling/testing">testing</Link> section more infomation
+Once that is done a coverage report can be generated by simply running: <InlineCode>npm run coverage</InlineCode>.
+
+<Flash scheme="yellow">
```

Comment:
I would be tempted to stick with the default theme.

Blue = info
Yellow = warning
Red = danger

These are the generally accepted semantics of those colours

Reply:
I saw this as a warning that the code coverage folder would need to be ignored so the choice of yellow was intentional. Are you thinking this is more an info flash?